dev/sequence_diagrams/on-click-post-word-from-undo.md

```diff
@@ -0,0 +1,64 @@
+```plantuml
```

Comment:
Make sure the changes you made here is applied in the PR https://github.com/ajktown/docs/pull/33

Reply:
Merge Order 
1. https://github.com/ajktown/docs/pull/33
2. https://github.com/ajktown/docs/pull/35